apps/pollbook/frontend/src/update_address_flow.tsx

```diff
@@ -101,14 +119,16 @@ function UpdateAddressScreen({
             address.streetName !== '' &&
             !isAddressValid && (
               <Callout icon="Danger" color="danger">
-                Invalid address. Make sure the street number and name match a
-                valid address for this jurisdiction.
+                Invalid address for {election.county.name}. Make sure the street
+                number and number match a valid address.
               </Callout>
             )}
           {isAddressInWrongPrecinct && (
             <Callout icon="Danger" color="danger">
-              This address is not in the current precinct. Voters can only have
-              their address changed to addresses within the configured precinct.
+              This address is associated with a different precinct,{' '}
+              {enteredPrecinct && enteredPrecinct.name}. Voters can only have
+              their address changed to addresses within the current precinct,{' '}
+              {configuredPrecinct && configuredPrecinct.name}.
```

Comment:
my copy was grammatically incorrect, it should be "...address changed to **an address** within..."